content/pt-br/docs/concepts/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Proxies no Kubernetes
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+Esta página descreve o uso de proxies com Kubernetes. 
+
+
+<!-- body -->
+
+## Proxies
+Existem vários tipos diferentes de proxies que você pode encontrar usando Kubernetes:
+
+
+1.  O [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - executa na máquina do usuário ou em um pod
+    - proxies de um host local endereçam para o servidor de API do kubernetes
+    - cliente para o proxy usa HTTP 
+    - proxy para o servidor de API usa HTTPS 
+    - localiza o servidor de API
+    - adiciona os cabeçalhos de comunicação.
```

Comment:
This whole list does not makes much sense. I think what we are trying to say here is how kubectl proxy works, so maybe the opening phrase should be "Quando o kubectl proxy é utilizado ocorre o seguinte:"

Reply:
Look into the new changes, is it more clear now?